src/ldp.rs

```diff
@@ -1653,10 +1653,6 @@ impl ProofSuite for JsonWebSignature2020 {
         extra_proof_properties: Option<Map<String, Value>>,
     ) -> Result<Proof, Error> {
         let algorithm = key.get_algorithm().ok_or(Error::MissingAlgorithm)?;
-        if algorithm != Algorithm::ES256 {
```

Comment:
Why wasn't this `RS256`? I'm confused.

Reply:
Previously this suite allowed only ES256 because that was the algorithm with which we were using this suite. I removed that check so it could support more JWS algorithms.
However, there should probably still be a check here, since JsonWebSignature2020 is supposed to support a limited set of JWS algorithms (e.g. not RS256): https://w3c-ccg.github.io/lds-jws2020/#jose-conformance